packages/sdk-js/README.md

```diff
@@ -4,86 +4,20 @@
 [![Lint and Test](https://github.com/KILTprotocol/sdk-js/workflows/Lint%20and%20Test/badge.svg)](https://github.com/KILTprotocol/sdk-js/actions/workflows/tests.yml)
 [![Compatible with latest dependencies](https://github.com/KILTprotocol/sdk-js/actions/workflows/tests-polkadot-deps.yml/badge.svg?event=schedule)](https://github.com/KILTprotocol/sdk-js/actions/workflows/tests-polkadot-deps.yml)
 
-Data sovereignty and interoperability
-
 # KILT JavaScript / TypeScript SDK
 
-The open-source KILT SDK is written in TypeScript and enables you to quickly and easily build dApps around new business use cases. KILT is a protocol for self-sovereign data and interoperability built on top of the permissionless KILT blockchain. The SDK provides collection of classes and methods you can utilize to interact with the KILT Protocol.
+The open-source KILT SDK is written in TypeScript and enables developers to quickly and easily build dApps around new business use cases.
+KILT is a protocol for self-sovereign data and interoperability built on top of the permissionless KILT blockchain.
+The SDK provides collection of classes and methods to interact with the KILT Protocol.
 
 - **Self-sovereign data.** Have ownership of your digital and analog identities, with control over who your users share that data with. Providing that extra layer of flexiblity and security.
 - **Interoperability.** Claim Types (CTYPEs) facilitate the adoption of standardised credential content structures.
 
-To learn more, checkout the KILT [Whitepaper](https://kilt.io/wp-content/uploads/2019/05/KILT-Whitepaper-v2019-May-28.pdf) or see our [how it works](https://kilt.io/kilt-data-sovereignty-and-interoperability/) video.
```

Comment:
Hmm, many people say, that they read the whitepaper and get a fairly good understanding from it, even if it is not up–to-date. Also the video is still relevant I think?

Reply:
I might agree on having people reading the whitepaper, although I think it is also kind of "dangerous". But is the SDK documentation a good place to have a reference to it? We even want to remove it from the docs main page.

Reply:
Same applies for the other things. People using the SDK should already be familiar with KILT. It feels like we are mixing the "concepts" and the "develop" ideas here as well.